src/geo/leaflet/leaflet-cartodb-vector-layer-group-view.js

```diff
@@ -27,6 +27,19 @@ var LeafletCartoDBVectorLayerGroupView = CartoDBd3Layer.extend({
       this.setCartoCSS(index, style);
     }, this);
 
+    layerModel.layers.bind('change:meta', function (child, meta) {
+      if (this.model.get('type') === 'namedmap') {
+        var index = child.collection.indexOf(child);
+        if (child.collection.at(0).get('type') === 'tiled') {
```

Comment:
I'm not sure where `child.collection` comes from (I don't see it in the Backbone docs), but we could use `layerModel.layers` instead, which surely doesn't have any `tiled` layer.